### DISCUSSION TITLE
Using Controller seems to require an extra click (lost focus?)

### DISCUSSION THREAD

**Discussion by skube** [react-hook-form/react-hook-form#2889] [Q&A] [answered]
I'm not sure if this is a bug or not but I can't figure out what's going on—and why it only affects Chrome (not Firefox or Safari)?

Here's [the Codesandbox](https://codesandbox.io/s/epic-test-issue-rpk3u?file=/src/App.tsx)

The issue is that I after choosing the **someone** radio, I have to double-click to activate/focus one of the A,B,C buttons.

In addition, the keyboard loses focus when choosing one of the A,B, C buttons with the keyboard.

Any insight or ideas to try would be appreciated!

**Comment by bluebill1049** [comment]
i suspect it's todo with `key={value}` change to something more unique?

**Reply by skube** [reply]
Thanks for responding! 

I've [updated the CSB](https://codesandbox.io/s/epic-test-issue-rpk3u?file=/src/App.tsx) to use slightly more unique keys (even though "A", "B", "C" are unique)—unfortunately same behaviour.

Also, I should note this is a reduced-test case example. On my actual code, the double-click issue presents on _any_ form input—even those without a `map` happening.

Any other ideas?

**Reply by bluebill1049** [reply]
Not on top of my head, I would suggest having a look at this example: https://codesandbox.io/s/react-hook-form-parse-and-format-textarea-furtc

**Reply by skube** [reply]
OK, I'll take a look and see if there is any hints there.

**Comment by skube** [comment]
I've also noticed something strange—perhaps related—using [another CSB](https://codesandbox.io/s/epic-test-issue-forked-vfj5z?file=/src/App.tsx) and RHF DevTools, you'll get strange results when clicking through the buttons.

**Reply by JeromeDeLeon** [reply]
The RHF version from the codesandbox you provided was already outdated though and I tried updating it to the latest version and seemed to solve the problem.

**Comment by skube** [comment, marked as answer]
FYI, finally got to the bottom of what was causing the issue. It seems the `watch` was the culprit. Using a state variable instead to manage things worked out much better. I'm using an older version of RHF, so it won't be an issue in later versions of RHF (as confirmed by @JeromeDeLeon)
